src/shrimps/views.py

```diff
@@ -38,11 +43,19 @@ def give_birth_to_shrimp(request):
     new_shrimp = Shrimp(name=name, size=size, weight=weight, color=color, is_farmed=is_farmed)
     new_shrimp.save()
 
-    return HttpResponse(
-         f"""A new baby shrimp named {name} is born, it weights {weight} g and measures {size} mm.""")
-
+    context = {
+        "shrimp": new_shrimp,
+    }
+    return render(request, template_name="detail.html", context=context)
 
 
+def shrimp_detail(request, shrimp_id):
+    """Display detail of shrimp."""
+    shrimp = Shrimp.objects.get(id=shrimp_id)
```

Comment:
You need to handle somehow the case where there is currently no shrimp in the database. You can either check it before you send that to your render function, or use the filtering, for example "Shrimp.objects.filter(id=shrimp_id).first()".